init.go

```diff
@@ -176,6 +181,14 @@ func init() {
 		logrus.Fatalf("Failed to detect hostname: %s", err)
 	}
 
+	if *kubeconfigF != "" {
+		data, err := ioutil.ReadFile(*kubeconfigF)
+		if err != nil {
+			logrus.Fatalf("Failed to read kubeconfig: %s", err)
+		}
+		Kubeconfig = string(data)
```

Comment:
The comment at line 52 says: `// Kubeconfig path to kubeconfig.` but we are storing the contents of the file here, as a string. The path is in the kingping flag `kubeconfigF`

Reply:
Thanks. Fixed comment.